pkg/services/postgresqldb/types_dbms_only.go

```diff
@@ -5,9 +5,15 @@ import "github.com/Azure/open-service-broker-azure/pkg/service"
 // ServerProvisioningParameters encapsulates non-senstivie PostgreSQL-specific
 // dbms provisioning options
 type ServerProvisioningParameters struct {
-	SSLEnforcement  string `json:"sslEnforcement"`
-	FirewallIPStart string `json:"firewallStartIPAddress"`
-	FirewallIPEnd   string `json:"firewallEndIPAddress"`
+	SSLEnforcement string         `json:"sslEnforcement"`
+	FirewallRules  []FirewallRule `json:"firewallRules"`
+}
+
+// FirewallRule describes a firewall rule to be applied to an DBMS
+type FirewallRule struct {
+	RuleName string `json:"ruleName"`
```

Comment:
Is the "rule" prefix on the field superfluous here as well?

Reply:
Probably! Let's get rid of it. I like it. Standby